facade/native/src/test/scala/AkkaConf.scala

```diff
@@ -0,0 +1,23 @@
+/* Copyright 2016 UniCredit S.p.A.
+*
+* Licensed under the Apache License, Version 2.0 (the "License");
+* you may not use this file except in compliance with the License.
+* You may obtain a copy of the License at
+*
+* http://www.apache.org/licenses/LICENSE-2.0
+*
+* Unless required by applicable law or agreed to in writing, software
+* distributed under the License is distributed on an "AS IS" BASIS,
+* WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+* See the License for the specific language governing permissions and
+* limitations under the License.
+*/
+object AkkaConf {
+
+  def basic =
+    io.Source.fromFile("facade/jvm/src/test/resources/akka.conf").mkString
```

Comment:
so, you added files under `facade/native` but you are using files from `facade/jvm` ? not sure why ...

Reply:
fixed